mono/eglib/gspawn.c

```diff
@@ -522,5 +519,9 @@ g_spawn_async_with_pipes (const gchar *working_directory,
 	if (standard_error)
 		*standard_error = err_pipe [0];
 	return TRUE;
-#endif
 }
+
+#endif // HAVE_G_SPAWN
+
+extern const char mono_squash_empty_file_warning_gspawn.
+extern const char mono_squash_empty_file_warning_gspawn = 0.
```

Comment:
Don't we have a macro for this? Or does that live in utils?

Reply:
There is a macro but I'd just as soon write it out.

There is a problem with this PR though..like..there is code kinda using spawn on Win32.
Kinda.

Reply:
I would prefer the macro...

Though if Windows code is using spawn then maybe the proper solution here is to just implement this for Windows? I don't think it's that complicated?

Reply:
Filling in the implementation would change the behavior, of the caller that is currently, I guess, expecting it to do nothing. Or maybe the call never materializes dynamically.